packages/react-resizable-panels/src/env-conditions/development.ts

```diff
@@ -0,0 +1 @@
+export const isDevelopment = true;
```

Comment:
There is a small caveat right now that you won't get auto-imports for this `#is-development` thing. There is a tracking issue about this [here](https://github.com/microsoft/TypeScript/issues/52460). Emma (Preconstruct's author) is already working on adding support for that - that will benefit everybody since `package.json#imports` are by no means Preconstruct's feature.

Reply:
Am I understanding you correctly to mean that e.g. VS Code auto-imports will stop working for the monorepo?

Reply:
Ir are you literally just saying that attempts to import `isDevelopment` will need to be manually written?

Reply:
> I think you're just saying that attempts to import isDevelopment will need to be manually written?

This. Once you add the import to the file then `isDevelopment` will appear in the suggestions with it - it's just about the auto-import thing (and it's a temporary situation, hopefully, since the support in the TS lang server for this will come in the future).